presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -110,6 +110,9 @@ public class ClientOptions
     @Option(name = "--debug", title = "debug", description = "Enable debug information")
     public boolean debug;
 
+    @Option(name = "--runtime-stats", title = "runtime stats", description = "Enable run time stats information")
```

Comment:
One final nit, please mention that this flag must be enabled in conjunction with the --debug flag.